.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
I didn't notice this file before can we remove it?  I think I prefer not to commit ide specific files into the source repo if you don't mind.